syft/pkg/cataloger/java/archive_parser_test.go

```diff
@@ -1159,6 +1224,20 @@ func Test_parseJavaArchive_regressions(t *testing.T) {
 	}
 }
 
+func assignParent(expectedPackages []pkg.Package) {
```

Comment:
nit suggestion: a little easier to understand, even if the caller was doing something akin to `assignParent(&tt.expectedPkgs[0], tt.expectedPkgs[1:]...)`

```suggestion
func assignParent(parent *pkg.Package, childPackages ...pkg.Package) {
```